### PR TITLE
Enhanced tabbed browsing

### PR DESCRIPTION
**Pull request by utkarshnath**
solved #1019
@aayusharora please check

### CODE REVIEW

**Comment by aayusharora** [comment]
@utkarshnath Please ask on issues before sending the PR.
I think somebody is already working on this.

**Comment by utkarshnath** [comment]
@aayusharora sorry , I'll take care of it in future

File: src/backend/templates/rooms.hbs [hunk 1]
```diff
@@ -120,14 +120,14 @@
 		             	    </p>
 		             	  {{/if}}
 		             	{{/speakers_list}}
-		             	<ul style="padding:0;font-size:15px;">
+                    <a href="javascript:;"><ul style="padding:0;font-size:15px;">
```
Comment (utkarshnath) [inline src/backend/templates/rooms.hbs]: @aayusharora I was trying to add a dummy link to each box as tabs can reach only those items which have some link in them

Comment (aayusharora) [inline src/backend/templates/rooms.hbs]: @utkarshnath Writing `<a href="javascript:;">`  is a bad practice
Check this http://stackoverflow.com/questions/7755088/what-does-href-expression-a-href-javascript-a-do 
You can return false by calling a dummy function.

**Comment by utkarshnath** [comment]
@aayusharora implemented the changes requested

**Comment by aayusharora** [comment]
Can you please squash the commits. Also , provide a live link of app demonstrating the progress @utkarshnath

**Comment by utkarshnath** [comment]
@aayusharora

ezgif com-video-to-gif

**Comment by aayusharora** [comment]
Thanks @utkarshnath

**Pull request merged**